core/src/main/scala/kafka/admin/PreferredReplicaLeaderElectionCommand.scala

```diff
@@ -67,7 +67,6 @@ object PreferredReplicaLeaderElectionCommand extends Logging {
       val preferredReplicaElectionCommand = new PreferredReplicaLeaderElectionCommand(zkUtils, partitionsForPreferredReplicaElection)
 
       preferredReplicaElectionCommand.moveLeaderToPreferredReplica()
-      println("Successfully started preferred replica election for partitions %s".format(partitionsForPreferredReplicaElection))
```

Comment:
We use `println` in CLI tools, so it doesn't seem right to replace this by a logging call.